app/models/conference.rb

```diff
@@ -26,7 +26,11 @@ class Conference < ActiveRecord::Base
   has_many :ticket_purchases, dependent: :destroy
   has_many :payments, dependent: :destroy
   has_many :supporters, through: :ticket_purchases, source: :user
-  has_many :tickets, dependent: :destroy
+  has_many :tickets, dependent: :destroy do
+    def find_registration_tickets
```

Comment:
Why not just calling this something like `for_registration?` then it will be called `tickets.for_registration?`